pkg/virt-controller/services/template.go

```diff
@@ -848,7 +848,8 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			RunAsUser:  &userId,
 			Privileged: &privileged,
 			Capabilities: &k8sv1.Capabilities{
-				Add: capabilities,
+				Add:  capabilities,
+				Drop: []k8sv1.Capability{CAP_NET_RAW},
```

Comment:
Can you please specify why an explicit drop is needed in the commit message? Other capabilities are just not added, but I guess this one is enabled by default with some CRI implementations.

Reply:
yes, this one may be enabled by default. Done.